regression-tests/pure2-bugfix-for-parameter-decl-list.cpp2

```diff
@@ -0,0 +1 @@
+main: (args,) = {}
```

Comment:
The test name should end in `-error.cpp2`, like the error Cpp2-time error tests.

Reply:
Oh, right. Thanks!